spec/lite_spec_helper.rb

```diff
@@ -57,6 +57,7 @@
 require 'support/sdam_monitoring'
 require 'support/crud'
 require 'support/command_monitoring'
+require 'support/cluster_tools'
```

Comment:
The tests in `spec_tests/uri_options/spec` rely on `ClusterTools` being present and will throw errors if run without this require.